common/setups/rasr/onnx_precomputed_hybrid_system.py

```diff
@@ -0,0 +1,109 @@
+import itertools
+from typing import Dict, List, Optional, Union
+
+from sisyphus import tk
+
+import i6_core.rasr as rasr
+import i6_core.returnn as returnn
+from i6_core.returnn.compile import TorchOnnxExportJob
+from i6_core.returnn.flow import make_precomputed_hybrid_onnx_feature_flow, add_fwd_flow_to_base_flow
+from i6_experiments.common.setups.rasr.hybrid_system import HybridSystem
+
+Path = tk.setup_path(__package__)
+
+
+class OnnxPrecomputedHybridSystem(HybridSystem):
+    """
+    System class for hybrid systems that train PyTorch models and export them to onnx for recognition. The NN
+    precomputed hybrid feature scorer is used.
+    """
+
+    def nn_recognition(
+        self,
+        name: str,
+        returnn_config: returnn.ReturnnConfig,
+        checkpoints: Dict[int, Union[returnn.Checkpoint, returnn.PtCheckpoint]],
+        train_job: Union[returnn.ReturnnTrainingJob, returnn.ReturnnRasrTrainingJob],
+        prior_scales: List[float],
+        pronunciation_scales: List[float],
+        lm_scales: List[float],
+        optimize_am_lm_scale: bool,
+        recognition_corpus_key: str,
+        feature_flow_key: str,
+        search_parameters: Dict,
+        lattice_to_ctm_kwargs: Dict,
+        parallelize_conversion: bool,
+        rtf: int,
+        mem: int,
+        epochs: Optional[List[int]] = None,
+        quantize_dynamic: bool = False,
+        needs_features_size: bool = True,
+        acoustic_mixture_path: Optional[tk.Path] = None,
+        nn_prior: bool = True,
```

Comment:
The default should not result in an `NotImplementedError`, so:


```suggestion
        nn_prior: bool = False,
```

Reply:
But the default should be to compute the prior. Can we use the `ReturnnComputePriorJobV2` here?
I'd rather have the PR merged so that the default throws an error.

Reply:
So that the user must make a decision to use the acoustic mixtures for the prior

Reply:
I don't mind. We should add support for NN prior either way. Maybe @Atticus1806 could merge his local code at some point? I'm inclinded to merge this PR before, though. And I'm fine with either default until then. @JackTemaki what do you think?

Reply:
What about no default for now?